plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -765,9 +683,9 @@ SecRule REQUEST_FILENAME "@rx /remote\.php/dav/calendars/(?:[^/]+/){2}[^/]+\.ics
         "t:none,\
         ctl:ruleRemoveById=950100"
 
-# Editing/deleting existing calendar appointments
+# Editing/deleting an existing calendar appointments
 # Calendar description can be anything
-SecRule REQUEST_FILENAME "@rx /apps/calendar/v[0-9]/appointment_configs/[0-9]+$" \
+SecRule REQUEST_FILENAME "@rx /apps/calendar/v[0-9.]/appointment_configs/[0-9]+$" \
```

Comment:
Is `v[0-9.]` correct? `v.` is a valid segment? If I had to guess, I'd say you're missing a `+` there: [v[0-9.]+`.